component/remote/http/http.go

```diff
@@ -160,25 +160,25 @@ func (c *Component) nextPoll() time.Duration {
 // not be held when calling. After polling, the component's health is updated
 // with the success or failure status.
 func (c *Component) poll() {
-	startTime := time.Now()
```

Comment:
I don't think it makes sense for the health time to be set when the poll _started_, rather than when it _finished._ I'm not sure why I did this initially, but I think it makes more sense with the update time being set to when the health was generated.